core/lib/makeswift/components/customer-group-slot/customer-group-slot.makeswift.ts

```diff
@@ -64,7 +64,16 @@ runtime.registerComponent(CustomerGroupSlot, {
         try {
           const data = await getAllCustomerGroups();
 
-          if (!data) return [];
+          if (!data) {
+            // nullish data means we don't have access to the customer groups API
+            return [
+              {
+                id: NO_GROUP_ID,
+                label: 'Setup needed. See docs.',
```

Comment:
Do we have a link to the docs we can provide?

Reply:
🍹  Given that the "builder user" is non-technical, should the message be something more akin to "Talk to your developer"?

Reply:
@chanceaclark We don’t have a public link to the docs yet, and even if we did, including it in the value dropdown wouldn’t be very helpful since it wouldn’t be fully visible or clickable. That said, we have a builder feature in the works that should provide a better experience here, I’ll follow up with another PR once it’s available.